flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/consumer/InputChannel.java

```diff
@@ -168,8 +176,7 @@ protected void notifyBufferAvailable(int numAvailableBuffers) throws IOException
      * <p>The queue index to request depends on which sub task the channel belongs to and is
      * specified by the consumer of this channel.
```

Comment:
```and is specified by the consumer of this channel.``` This java doc seems out of date. Maybe delete this java doc and add the java doc to the ```consumedSubpartitionIndex``` filed.

Reply:
Done